README.md

```diff
@@ -1,2 +1,32 @@
-# SSPINN
-Spectral Structure Prediction In Neural Networks
+![SSPINN Logo](./docs/images/SSPINN_Logo.png)
+
+[![Build Status](https://travis-ci.org/awild82/SSPINN.svg?branch=master)](https://travis-ci.org/awild82/SSPINN)[![Coverage Status](https://coveralls.io/repos/github/awild82/SSPINN/badge.svg?branch=master)](https://coveralls.io/github/awild82/SSPINN?branch=master)
+SSPINN is a software package meant to predict the structure of organic molecules based on experimentally obtained spectral data. Based on a user's input data, the SSPINN neural network will generate a connectivity matrix which predicts the bonding behavior between atoms in the user's molecule. This connectivity matrix will then be used to generate a graphical depiction of the predicted molecular structure. 
+
+## Software Requirements
+* [Python3](url)
+* [Keras](url)
+* [Networkx](url)
+    
+## Installation Instructions
+1. Clone git repository
+    *`git clone https://github.com/awild82/SSPINN.git`
+2. Install SSPINN by running setup file
+    *`./setup.sh`
+    
+## Structure Prediction
+1. Format <sup>13</sup>C data according to template in `SSPINN/docs/specs/nn_input.md`
+2. Save as .txt file in sspinn directory
+3. In terminal, run `python -m sspinn <nmr file>`
+
+## Testing
+1. Open SSPINN directory in terminal
+2. Run `nosetests`
+    * If nosetests is not already installed run `pip install nosetests`
+    
+## Training
+
+## Contributing
```

Comment:
* Something about reporting issues please (e.g. "Found an issue in our code? Great! Let us know by reporting an issue)
* Point to docs/developing.md for developing flow and style guidelines
* Something about letting us know if they want to contribute data